api/src/main/java/com/minirogue/api/media/MediaEnums.kt

```diff
@@ -47,9 +47,9 @@ enum class MediaType(val legacyId: Int) {
     @SerialName("Audiobook")
     AUDIOBOOK(14);
 
-    fun getSerialname() = MediaType.serializer().descriptor.getElementName(ordinal)
+    fun getSerialName() = MediaType.serializer().descriptor.getElementName(ordinal)
     companion object {
-        fun getFromLegacyId(legacyId: Int): MediaType? = values().firstOrNull { it.legacyId == legacyId }
+        fun getFromLegacyId(legacyId: Int): MediaType? = entries.firstOrNull { it.legacyId == legacyId }
```

Comment:
Fixed this while I was in this file. Most of the usages of `Enum.values()` in this project should be updated in a future PR.